GUI/widgets/plot_buttons.py

```diff
@@ -23,8 +24,46 @@ def initUI(self):
         self.example_button.setText("Example Button")
         self.example_button.setStyleSheet("font-family: Helvetica; font-size: 14px; background-color : #2B26c1")
 
+        # Add the widget for resizing the plot axes
+        resize_input_widgets = QSplitter(Qt.Vertical)
+        row1 = QSplitter(Qt.Horizontal)
+        row2 = QSplitter(Qt.Horizontal)
+        row3 = QSplitter(Qt.Horizontal)
+
+        self.textbox1 = QLineEdit(self)
+        self.textbox1.resize(100,100)
+        self.textbox1.setStyleSheet("font-family: Helvetica; font-size: 14px; background-color : #2B26c1")
+
+        self.textbox2 = QLineEdit(self)
+        self.textbox2.resize(100,100)
+        self.textbox2.setStyleSheet("font-family: Helvetica; font-size: 14px; background-color : #2B26c1")
+
+        self.textbox3 = QLineEdit(self)
+        self.textbox3.resize(100,100)
+        self.textbox3.setStyleSheet("font-family: Helvetica; font-size: 14px; background-color : #2B26c1")
+
+        self.textbox4 = QLineEdit(self)
+        self.textbox4.resize(100,100)
+        self.textbox4.setStyleSheet("font-family: Helvetica; font-size: 14px; background-color : #2B26c1")
```

Comment:
i think for these as well, the stylesheets can be applied from external css file

Reply:
yes definitely, I'll just batch everything with ticket 237